### DISCUSSION TITLE
Hard to read colors of the status window

### DISCUSSION THREAD

**Discussion by bconvens** [ctu-mrs/mrs_uav_system#148] [Q&A] [answered]
Hello,
I noticed the status window changed to blue.
I find it very hard to read now.
Is there a way to change it back to what it was before?
Thanks!

**Comment by petrapa6** [comment, marked as answer]
@DanHert Is there a way to change the colors?

**Reply by DanHert** [reply]
Hi, the blue color was introduced as default to help people with color blindness (they can confuse red and green).
To change back to the green color, just set colorblind mode to False in the mrs_uav_status config.

`colorblind_mode: False # Set True to change green color to blue
` 
By default, the config file used is located in mrs_uav_status/config/default.yaml
